src/components/Modal/ModalHeader.tsx

```diff
@@ -0,0 +1,46 @@
+/** @jsx jsx */
+import { css, jsx } from '@emotion/core'
+
+import { Color } from 'csstype'
+import { Theme, useTheme } from '../../styles'
+import { Heading } from '../Heading'
+import { ModalCloseButton } from './ModalCloseButton'
+
+export type ModalTitleType = string | JSX.Element
+
+export interface ModalHeaderProps {
+  title: ModalTitleType
+  backgroundColor?: Color
```

Comment:
I think you can expose the "style" prop so you pass the background color inside the css.